src/shared/Core/Core.csproj

```diff
@@ -21,12 +21,13 @@
     <PackageReference Include="Microsoft.Identity.Client" Version="4.52.0" />
     <PackageReference Include="Microsoft.Identity.Client.Extensions.Msal" Version="2.28.0" />
     <PackageReference Include="System.CommandLine" Version="2.0.0-beta1.21216.1" />
-    <PackageReference Include="Avalonia" Version="0.10.18" />
-    <PackageReference Include="Avalonia.Desktop" Version="0.10.18" />
+    <PackageReference Include="Avalonia" Version="11.0.0-preview6" />
```

Comment:
Wow! Big jump.

Reply:
Yeah they moved from `0.10` to dropping the `0.` prefix with v11